### PR TITLE
[7.x] Switch to Symfony 5

### PR DESCRIPTION
**Pull request by GrahamCampbell**
Pairs with https://github.com/laravel/framework/pull/30610.

### CODE REVIEW

**Comment by taylorotwell** [comment]
@GrahamCampbell Laravel 7.0 is no longer installable because of complaints about Tinker not supporting Symfony 5.

**Comment by GrahamCampbell** [comment]
@taylorotwell Yes, I'm aware of this, and I raised this with tinker a few days ago. It will be fixed in the coming weeks. :)

**Comment by GrahamCampbell** [comment]
In the meanwhile, Laravel 7 can still be installed without tinker.

**Comment by GrahamCampbell** [comment]
@taylorotwell We can track the tinker stuff at https://github.com/laravel/tinker/pull/79. :)

File: config/session.php [hunk 1]
```diff
@@ -166,7 +166,7 @@
     |
     */
 
-    'secure' => env('SESSION_SECURE_COOKIE', false),
+    'secure' => env('SESSION_SECURE_COOKIE', null),
```
Comment (mathieutu) [inline config/session.php]: Hi @GrahamCampbell why did you set null here, as env default parameter is by default null?

File: config/session.php [hunk 2]
```diff
@@ -194,6 +194,6 @@
     |
     */
 
-    'same_site' => null,
+    'same_site' => 'lax',
```
Comment (mathieutu) [inline config/session.php]: Hi @GrahamCampbell why did you change null to lax here? It changes the behavior but Symfony 5 does not need this. (We totally can pass null to Cookie).

Comment (GrahamCampbell) [reply config/session.php]: Because we explicitly decided we wanted to change the default in Laravel.

Comment (GrahamCampbell) [reply config/session.php]: And actually, having it as `null` broke our test suite switching from Symfony 4.3 to 5.0, so they must have changed something...

Comment (KennedyTedesco) [reply config/session.php]: Just a side note: lax will be the default on Chrome. https://www.chromestatus.com/feature/5088147346030592
